### DISCUSSION TITLE
Wrong "Inverter Sleep Time" -> No AC Spot Data

### DISCUSSION THREAD

**Discussion by Hofstelle31** [SBFspot/SBFspot#677] [Q&A]
Hi everybody,
One of my inverters (STP12000TL-10) has been having problems with the "Inverter Wake-Up time" or "Inverter Sleep Time" for a few days now.  The "Current Inverter Time" is correct and can also be adjusted using -settime if there are deviations.  Unfortunately, I have not yet been able to find out which criteria "wake-up time" and "sleep time" are generated and whether any can be corrected.  I can see from the log file that DC spot data is shown when the wake-up or sleep time has passed, but AC spot data is not.  Who can help?

![Bildschirmfoto zu 2023-10-29 17-15-27](https://github.com/SBFspot/SBFspot/assets/149302666/ae6e1a42-2bd6-4b6d-83de-4d15e735c6a4)

**Comment by sbf-** [comment]
> I can see from the log file that DC spot data is shown when the wake-up or sleep time has passed, but AC spot data is not

At that moment, the grid relay is still open (no production) so there are no AC values yet

**Reply by Hofstelle31** [reply]
The mains relay is open, as far as correct.  But why aren't the "Sleep Time" and "Wake-Up Time" set correctly to switch on the mains relay?  To explain: There are other inverters of the same type, everything is ok there.  To understand the function, I would like to know where the two times are generated: Is the communication module in the inverter responsible for this (or another component in the inverter)?  Does SBFSpot only show the times the inverter delivers?  I can't find any way to influence the times in Sunny Explorer.  Sorry about the many questions and thanks for your reply.

**Reply by sbf-** [reply]
Sleep and wake-up times aren't provided by inverter, but determined by SBFspot.
You can't change these times. The production starts/stops (relay closed/opened) depending of the amount of light on the panels.
